Tests/Kernel/TestMemoryDeviceMmap.cpp

```diff
@@ -23,7 +23,11 @@ static ALWAYS_INLINE bool mem_chunk(int fd, u64 base, u64 length)
 {
     u64 mmoffset = base % sysconf(_SC_PAGESIZE);
     void* mmp = mmap(NULL, mmoffset + length, PROT_READ, MAP_SHARED, fd, base - mmoffset);
-    return mmp != MAP_FAILED;
+    bool success = (mmp != MAP_FAILED);
+    if (munmap(mmp, mmoffset + length) < 0) {
```

Comment:
We probably shouldn't try to munmap if the mmap failed.

Reply:
Right. I'll fix it right away :)